app/src/pages/Lobby/index.tsx

```diff
@@ -36,9 +38,56 @@ const useStyles = makeStyles((theme: Theme) =>
 
 function ShareSection() {
   const currentGame = React.useContext(CurrentGameContext)
+  const [copyButtonClicked, setCopyButtonClicked] = React.useState(false)
+
+  React.useEffect(() => {
+    let timeout: NodeJS.Timeout
+    if (copyButtonClicked) {
+      timeout = setTimeout(() => {
+        setCopyButtonClicked(false)
+      }, 1000)
+    }
+    return () => timeout && clearTimeout(timeout)
+  }, [copyButtonClicked])
+
   return (
     <Grid item>
-      {`Share the code with everyone playing`}
+      {`Share your link with everyone playing`}
+      <Grid container spacing={2} style={{ paddingTop: 8, paddingBottom: 8 }}>
```

Comment:
for future - one solution here to stick w/the framework is wrappign w/ `<Box pt={1} pb={1}>`

cc: @wtachau